app/services.py

```diff
@@ -52,3 +55,47 @@ def delete_user(db: Session, profile_id: str):
     db.commit()
     return f"User {profile_id} has been deleted"
 
+
+def save_onesite_data(db: Session, file: UploadFile = File(...)):
+    filepath = os.path.realpath(os.path.join("fixtures", file.filename))
+    excel_workbook = xlrd.open_workbook(filepath)
+    all_worksheet = excel_workbook.sheet_by_name("All")
+    curr_worksheet = excel_workbook.sheet_by_name("Curr")
+    curr_row = 3
+
+    while curr_row < curr_worksheet.nrows-1:
+        curr_row += 1 
+        row = curr_worksheet.row_values(curr_row)
+        db_OneSite = OneSiteData(
+            property=row[0],
+            ReshID= row[1],
```

Comment:
just change these keyword arguments to snake_case instead of CamelCase and you should be good to go 👍